requirements.txt

```diff
@@ -1,4 +1,4 @@
 boto3>=1.10.0
-cryptography>=2.5.0
+cryptography>=2.5.0,<3.3.2
```

Comment:
Issue/Blocking: What? 
You convinced me that we were not going to pin `pyca`.
This looks like we are pinning `pyca`.